src/layout/section/highlightSection/HighlightSection.tsx

```diff
@@ -1,27 +1,100 @@
-import React from 'react'
+import React, { useEffect, useRef, useState } from 'react'
+import uniqueId from 'lodash.uniqueid'
 
-import { SectionContentSize } from '../../../layout/section/baseSection'
-import { StyledHighlightSection } from './HighlightSection.style'
+import { ItemChoiceProps } from '../../../itemChoice'
+import { Col, Grid, HighlightSectionElements } from './HighlightSection.style'
 
+export type HighlightItemsType = ItemChoiceProps &
+  Readonly<{
+    hidden: boolean
+  }>
+
+type GridListItemsProps = { items: Array<HighlightItemsType> }
+const GridListItems = ({ items }: GridListItemsProps) => {
+  const listItems = items.map(({ id, label, data, hidden, href }) => (
+    <Col key={id} hidden={hidden}>
+      <HighlightSectionElements.Item label={label} data={data} href={href} />
+    </Col>
+  ))
+  return <Grid>{listItems}</Grid>
+}
+
+type HighlightContentItemsProps = {
+  heading: string
+  items: Array<HighlightItemsType>
+}
+const HighlightContentItems = ({ heading, items }: HighlightContentItemsProps) => (
+  <HighlightSectionElements.Article>
+    <HighlightSectionElements.Title as="h2">{heading}</HighlightSectionElements.Title>
+    <GridListItems items={items} />
+  </HighlightSectionElements.Article>
+)
+
+const DEFAULT_ITEMS_SIZE = 3
+
+export type HighlightsType = { heading: string; items: Array<HighlightItemsType> }
 export type HighlightSectionProps = Readonly<{
   className?: string
-  contentClassName?: string
-  children: React.ReactNode
+  highlights: { axes: HighlightsType; destinations?: HighlightsType }
+  toggle: { on: string; off: string }
 }>
 
-/**
- * A specialized section with an highlighting background color.
- */
-export const HighlightSection = (props: HighlightSectionProps) => {
-  const { className, children, contentClassName } = props
+const setFocus = (collapsed: Boolean) => {
+  const collapsibleRegionWrapper = useRef()
+
+  const focusCollapsibleRegion = (): void => {
+    const collapsibleRegion = collapsibleRegionWrapper.current as HTMLDivElement
+    collapsibleRegion.focus()
+  }
+
+  useEffect(() => {
+    if (!collapsed) {
+      focusCollapsibleRegion()
+    }
+  }, [collapsed])
+
+  return [collapsibleRegionWrapper]
+}
+
+export const HighlightSection = ({ highlights, toggle, className }: HighlightSectionProps) => {
+  const [collapsed, setCollapsed] = useState(true)
+  const { axes, destinations } = highlights
+
+  const displayedItems = axes.items.map((item, index) => ({
+    ...item,
+    hidden: collapsed && index >= DEFAULT_ITEMS_SIZE,
+  }))
+
+  // Set Focus
+  const [collapsibleRegionWrapper] = setFocus(collapsed)
+  const collapsibleRegionId = uniqueId('region-')
```

Comment:
Thanks, I forgot the `uniqueId` existed, will need to use it more

Reply:
yeah! it's a great thing to compose our lists!